packages/eslint-config/index.js

```diff
@@ -48,7 +44,7 @@ module.exports = {
     // Allow reassigning props of objects, e.g. `foo.bar = 2`
     'no-param-reassign': ['error', {
       props: false
-    }]
+    }],
```

Comment:
Drop this comma to fix one of the linting errors. The rest of the linting errors are from those issues with `import` I believe.

(xo has some weird rules, I mostly use it for small projects because you don't have to set anything up.)

Reply:
This becomes unneeded since I added keys after